data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, TRUE, 10.23);
```

Comment:
[OPTIONAL]
- [ ] Since the `id` was already set as `SERIAL`  in the animals' table, there's no need to specify it value in this specific case since it's added automatically by `pglsql`
```sql
This could work
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true`, '10.23');

```